src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css

```diff
@@ -231,8 +231,6 @@
 .monaco-workbench .part.editor > .content .gettingStartedContainer .gettingStartedSlideCategories .featured-icon {
 	font-size: 20px;
 	padding-right: 8px;
-	position: relative;
```

Comment:
Is this change needed?

Reply:
This property can be deleted or not since using `position: relative` without `top: 3px` works the same as `position: static` (default value) for the element itself (https://stackoverflow.com/a/13394093)